device/inventec/x86_64-inventec_d6556-r0/INVENTEC-D6556/td3-d6556-48x25G-8x100G.config.bcm.inventec

```diff
@@ -0,0 +1,287 @@
+### fix for sonic
+cancun_dir=/usr/share/sonic/hwsku/
+led_fw_dir=/usr/share/sonic/platform/
+ptp_ts_pll_fref=50000000
+ptp_bs_fref_0=50000000
+ptp_bs_fref_1=50000000
+ifp_inports_support_enable=1
+### end fix
```

Comment:
we need remove inventec suffix so that we can check if all attribute in the permitted list.

Reply:
#1 For inv_pthread.c 
the kernel thread is used to check the PSU, FAN and to blink the healthly led for indicating the current system status. It is better to be implemented as a C code because the behavior of the healthy led is defined in the hardware spec of the platform. Also, it is better not to run them in user space in case of the process is killed by some unknown reason.

#2 for *.inventec and *.bin file. Let me remove them because it is for TD3 which is not supported in  current SAI yet.

Reply:
on the opposite, it is better to run them in user space as errors will not cause the system to crash. Meanwhile, any thing can be killed, even the quagga. but in user space, we can easier monitor those process to make sure they are running.

Reply:
In the past, the healthy led is done in FW. But, since the system is more and more complicated, some of the behaviors are moved from FW to linux kernel thread because it could handover the control as soon as possible (the kernel thread is loaded in rcS) and ease for integration (some commerical NOS don't like to have user-space application). Also, its behavior should be the same on different platforms (to ease operators to identify the issue), so that it is better to be implemented by C/C++ and compiled, not python script. Please consider to have kernel thread in this version. We will study how to move it the userspace.